### PR TITLE
docs: fix date and time type

### LINKED ISSUE #2172: docs: the introduction of data type timestamp is not compeleted
https://openmldb.ai/docs/en/main/reference/sql/data_types/date_and_time_types.html
https://openmldb.ai/docs/zh/main/reference/sql/data_types/date_and_time_types.html

### PR DESCRIPTION
**Pull request by vagetablechicken**
Closes #2172

### CODE REVIEW

**Review by lumianph — APPROVED** [review]

**Comment by lumianph** [comment]
@michelle-qinqin 
you may also take a look

File: docs/zh/reference/sql/data_types/date_and_time_types.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-OpenMLDB支持日期类型`DATE`和时间戳`TIMESTAMP`
+OpenMLDB支持日期类型`DATE`和时间戳`TIMESTAMP`。
 
 每个时间类型有一个有效值范围和一个NULL值，当指定不合法不能表示的值时使用NULL值。
 
 | 类型      | 大小 (bytes) | 范围                                                         | 格式            | 用途                     |
 | :-------- | :----------- | :----------------------------------------------------------- | :-------------- | :----------------------- |
 | DATE      | 4            | 1900-01-01 ~                                                 | YYYY-MM-DD      | 日期值                   |
-| TIMESTAMP | 8            | 1970-01-01 00:00:00/2038结束时间是第 **2147483647** 秒，北京时间 **2038-1-19 11:14:07**，格林尼治时间 2038年1月19日 凌晨 03:14:07 | YYYYMMDD HHMMSS | 混合日期和时间值，时间戳 |
+| TIMESTAMP | 8            | 从 ‘1970-01-01 00:00:01.000000’ UTC 到 ‘2038-01-19 03:14:07.999999’(2038-01-19 11:14:07.999999 GMT) | 在线: int64, 离线: int64 或 'yyyy-MM-dd'T'HH:mm:ss[.SSS][XXX]' | 混合日期和时间值，时间戳 |
```
Comment (michelle-qinqin) [inline docs/zh/reference/sql/data_types/date_and_time_types.md]: 请问命令行的离线模式下， 也可以使用'yyyy-MM-dd'T'HH:mm:ss[.SSS][XXX]'格式输入嘛。之前好像说是sdk用object，cli需要转成毫秒时间戳

File: docs/en/reference/sql/data_types/date_and_time_types.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
-Each time type has a valid range of values ​​and a NULL value. The NULL value is used when specifying an invalid value that cannot be represented。
+Each time type has a valid range of values ​​and a NULL value. The NULL value is used when specifying an invalid value that cannot be represented.
 
 | Type      | Size (bytes) |   Scope                                                         | Format            | Use                     |
 | :-------- | :----------- | :----------------------------------------------------------- | :-------------- | :----------------------- |
 | DATE      | 4            | 1900-01-01 ~                                                 | YYYY-MM-DD      | Date Value                   |
-| TIMESTAMP | 8            | End Time is 1970-01-01 00:00:00/2038 **2147483647** Second，Beijing time **2038-1-19 11:14:07**，GMT January 19, 2038 Early Morning 03:14:07 | YYYYMMDD HHMMSS | Mixed Date and Time Value, Timestamp |
+| TIMESTAMP | 8            | From ‘1970-01-01 00:00:01.000000’ UTC to ‘2038-01-19 03:14:07.999999’(2038-01-19 11:14:07.999999 GMT) | online: int64, offline: int64 or 'yyyy-MM-dd'T'HH:mm:ss[.SSS][XXX]' | Mixed Date and Time Value, Timestamp |
```
Comment (aceforeverd) [inline docs/en/reference/sql/data_types/date_and_time_types.md]: Timezone is default to UTC-8 even just write int64 as unix epoch time. This is a known issue.

Comment (vagetablechicken) [reply docs/en/reference/sql/data_types/date_and_time_types.md]: ok

File: docs/zh/reference/sql/data_types/date_and_time_types.md (hunk 1 above)
Comment (vagetablechicken) [reply docs/zh/reference/sql/data_types/date_and_time_types.md]: CLI 离线并不支持insert

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**